### PR TITLE
feat(country-page): init template - INNO-413

### PR DESCRIPTION
**Pull request by emeryro**
Implementation of template "Country page"
+ update of several component, to make them usable as composition

### CODE REVIEW

File: framework/components/indicators/_indicators.scss [hunk 1]
```diff
@@ -11,7 +11,8 @@
   border: 1px solid #fff;
   color: #fff;
   height: 10em;
-  margin: 0 auto;
+  margin-left: auto;
```
Comment (yhuard) [inline framework/components/indicators/_indicators.scss]: I like that 👍

File: framework/components/news/_news.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@
   &__image {
     margin: 0 auto;
     max-width: 20em;
+    transition: border-radius 0.2s;
```
Comment (yhuard) [inline framework/components/news/_news.scss]: I don't like that 👎 Whenever I see `transition` followed by anything but `opacity` or `transform`, it raises a warning in my mind

File: framework/base/_content.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+a,
```
Comment (yhuard) [inline framework/base/_content.scss]: We may need to talk about this 😉

**Comment by yhuard** [comment]
In a nutshell: 
- add padding on main links
- prevent leaflet from capturing the scroll (very annoying)

**Comment by emeryro** [comment]
Padding has been added to links.
Map scroll will be handled in another task related to map improvements (INNO-417)

**Pull request merged**